libraries/libraries.mk

```diff
@@ -110,7 +110,7 @@ $(BSG_PLATFORM_PATH)/libbsg_manycore_runtime.so.1.0: INCLUDES :=
 
 include $(BSG_PLATFORM_PATH)/library.mk
 
-$(LIB_OBJECTS) $(LIB_OBJECTS_CUDA_POD_REPL) $(LIB_OBJECTS_REGRESSION): INCLUDES := -I$(LIBRARIES_PATH)
+$(LIB_OBJECTS) $(LIB_OBJECTS_CUDA_POD_REPL) $(LIB_OBJECTS_REGRESSION): INCLUDES += -I$(LIBRARIES_PATH)
```

Comment:
This should remain =, so that it overrides previous definitions

Reply:
I wanted to add includes specific to the platform to build the libraries as well. I either need to move the include statement or I had to make it +=. Is there any solution you particularly prefer from the two I suggested or should I rethink how I handle this?

Reply:
I don't think I understand what you goal is

Reply:
These libraries should be platform agnostic. Is there something you are changing that affects that?

Reply:
Newlib and glibc differ in where they have some files stored. For example, in newlib features.h is in include/sys/features.h and in glibc it is in include/features.h. 

Since, we are compiling with newlib now, to make these files visible I have a software folder in the platform that has some headers and source files that add the missing pieces required for a successful compile. I need to include the headers for compile. I have that INCLUDE statement in the platform-specific library make fragment. I need to make that visible in the platform independent library make fragment as well. 

There is absolutely NO change to the code in the library.

Reply:
Reverted this back. Changed to suggestion given during meeting.